pkg/attribute/attribute.go

```diff
@@ -321,8 +321,14 @@ func (d Dictionary) GenerateTableInHTML() string {
 	<td>%s</td>
 	<td>%s</td>
 </tr>`
+
+		var typ interface{} = desc.typ
+		if typ == nil {
+			typ = "undetermined"
```

Comment:
Should "undetermined" be consistent with "unknown" in the codebase?

Reply:
Done.